.github/workflows/pull-request.yml

```
@@ -0,0 +1,666 @@
+    if: ${{ github.actor != 'dependabot[bot]' && !contains(github.event.pull_request.labels.*.name, 'ignore-sensitive-files-pr') }}
+    name: Checks if sensitive files have been changed without authorization
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0  # Fetch all history for all branches and tags
+
+      - name: Get Changed Unauthorized files
+        id: changed-unauth-files
+        run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2086:info:10:52: Double quote to prevent globbing and word splitting

(shellcheck)